adagrams/game.py

```diff
@@ -1,11 +1,149 @@
+import random
+
 def draw_letters():
-    pass
+    """
+    input: nothing
+    output: 10 random tiles in a list randomly taken from alphabet dictionary.
+    """
+    allletters = []
+    alphabet = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+    }
+    #expanded dictionary per numbers of letters
+    for letter, number in alphabet.items():
+        for i in range(number):
+            allletters.append(letter)
+    
+    #picking 10 random letters from expanded dictionary
+    gameletters = []
+    for letter in range(10):
+        number = random.randint(0, 27)
+        random_letter = allletters[number]
+        gameletters.append(random_letter)
+        allletters.remove(random_letter)
```

Comment:
On L45, the code chooses a random number between 1 to 27 and L36 indexes into that position for `allletters`.

Based on L38-40, this is our `allletters`:
```
allletters = ['A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'B', 'B', 'C', 'C', 'D', 'D',
 'D', 'D', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'F', 'F', 'G', 'G',
  'G', 'H', 'H', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'J', 'K', 'L', 'L', 'L', 'L', 'M', 'M', 
  'N', 'N', 'N', 'N', 'N', 'N', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'P', 'P', 'Q', 
  'R', 'R', 'R', 'R', 'R', 'R', 'S', 'S', 'S', 'S', 'T', 'T', 'T', 'T', 'T', 'T', 'U', 'U', 
  'U', 'U', 'V', 'V', 'W', 'W', 'X', 'Y', 'Y', 'Z']
```

Instead of randomly choosing from all possible letters, we only choose from the first 27 items in the list. That means we will never get values towards the end of our list. If we playtest the code, we see that our letters are suspiciously `A` , `B`, `C`, `D`, and `E` heavy. Remember to not just rely on our provided tests but run your code manually to make sure your logic is solid!